R/DataDiscoveryRetrieval.R

```diff
@@ -73,8 +76,8 @@ TADAdataRetrieval <- function(statecode = "null",
                               siteType = "null",
                               characteristicName = "null",
                               sampleMedia = "null",
-                              ProjectIdentifier = "null",
-                              OrganizationIdentifier = "null",
+                              project = "null",
+                              organization = "null",
```

Comment:
Same comment for OrganizationIdentifier (mention here in documentation & also in vignette?)

Reply:
See above!